### PR TITLE
8.0-demo-data

### PR DESCRIPTION
**Pull request by DaeronAlagos**
Adds account.invoice demo data for MTD module

### CODE REVIEW

File: account_mtd_vat/demo/account_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <field name="invoice_id" ref="customer_invoice_1"/>
+            <field name="account_id" model="account.account" search="[('code', '=', '400000')]"/>
+            <field name="uos_id" ref="product.product_uom_unit"/>
+            <field name="quantity" eval="1.0" />
+            <field name="price_unit" eval="320.0" />
+            <field name="price_subtotal" eval="320.0" />
+            <field name="discount" eval="50"/>
+            <field name="name">iPad Mini</field>
+            <field name="product_id" ref="product.product_product_6"/>
+            <field name="invoice_line_tax_id" model="account.tax" search="[('description', '=', 'ST11')]"/>
```
Comment (acela82) [inline account_mtd_vat/demo/account_invoice.xml:27]: Is it safer to use field name="invoice_line_tax_id" ref="account.tax_name" -- Taxes has xml_id, i dont know now which

Comment (DaeronAlagos) [reply account_mtd_vat/demo/account_invoice.xml:27]: That would be preferred, but Odoo 8 doesn't have the useful method of _create_records_with_xmlid() that exists in 13.0

There is an xmlid for ST11 on account.tax.template, but nothing on account.tax after installing and configuring the account module.

Comment (DaeronAlagos) [reply account_mtd_vat/demo/account_invoice.xml:27]: Had a look to see if I could refine the search to use the tax_template_id, but that's not viable either.

Comment (acela82) [reply account_mtd_vat/demo/account_invoice.xml:27]: Sorry , for taking a lot your time. :+1:

**Review by acela82 — APPROVED** [review]

**Pull request merged**